tests/components/header.pug

```diff
@@ -570,3 +571,146 @@ block content
+              path(d='M106.330232 1.71966316c0-.94108421-.572874-1.47287369-1.677516-1.47287369H89.1060842c-1.1043263 0-1.6771789.53178948-1.6771789 1.47287369v1.42565263l17.2648737-.0345579c1.063579-.00001052 1.636453-.49107368 1.636453-1.39109473', fill='#0C9ED9')
+          span.chi-header__title.-d--none.-d-sm--flex [Portal Name]
+        .chi-header__start
+          chi-button#toggle-nav(variant="flat", size="lg" type="icon")
+            chi-icon(icon="menu")
+        .chi-header__end
+          div.-align-items--center.-d--none.-d-lg--flex
+            <chi-dropdown class="enterprise-id-switcher" button="Enterprise ID: 1234567" size="sm" select-mode> <!-- or select-mode="single" -->
+              <div slot="menu-header">Enterprise ID</div>
+              <div slot="menu-header">Enterprise Name</div>
+              <a class="chi-dropdown__menu-item -active-item" slot="menu"><div>1234567</div><div>Acme Incorporated</div></a>
+              <a class="chi-dropdown__menu-item" slot="menu"><div>1234568</div><div>Acme Incorporated</div></a>
+              <a class="chi-dropdown__menu-item" slot="menu"><div>1234569</div><div>Acme Incorporated</div></a>
+              <a class="chi-dropdown__menu-item" slot="menu"><div>1234560</div><div>Acme Incorporated</div></a>
+              <chi-link slot="menu-footer">Manage My Enterprises</chi-link>
+            </chi-dropdown>
+            .chi-divider.-vertical
+            .chi-dropdown.-d--none.-d-lg--flex
+              <chi-dropdown position='bottom' button='Menu' variant='flat'>
+                <a class='chi-dropdown__menu-item' href='#' slot="menu">
+                  chi-icon(icon="user")
+                  | Item 1
+                </a>
+                <a class='chi-dropdown__menu-item' href='#' slot="menu">
+                  chi-icon(icon="login")
+                  | Item 2
+                </a>
+              </chi-dropdown>
+
+  h3 Test Portal Header with Impersonation Bar
+  .test-header-portal-header-bar-narnia.-w--100
+    header.chi-header.-portal.-impersonation.-lg
+      .chi-impersonation-bar
+        .chi-impersonation-bar__content
+          span.chi-impersonation-bar__label Logged in as:
+          strong.chi-impersonation-bar__username User
+          div.chi-impersonation-bar__divider.chi-divider.-inverse.-vertical.-h--75
+          button.chi-button.-flat.-light.-xs.-icon.-no-hover(id="button-logout", aria-label="Log out", data-tooltip="Log out", data-position="bottom")
+            .chi-button__content
+              i.chi-icon.icon-logout(aria-hidden="true")
```

Comment:
For the impersonation bar, there’s a few minor visual changes needed to match the spec:

1. Change line-height of `.chi-impersonation-bar__label` and `.chi-impersonation-bar__username` to `1rem` (`$line-height-sm`)
2. Change the before and after pseudo element height from `1.5rem` to `1.25rem`.
3. Apply a `-b--0` class to the logout flat icon button to further reduce size without having to define extra css code.
4. Change padding of `.chi-impersonation-bar__content` from `0 0.5rem` to `0 2rem`
5. Change label text from `Logged in as:` to `Signed in as:`